README.md

```diff
@@ -638,6 +640,53 @@ handle dependencies and CLDR loading manually yourself.
   [Read more...](doc/error/e-unsupported.md)
 
 
+## Contribution
```

Comment:
Leaving it as is for now, but we could definitely reevaluate and change it later.